lightwood/helpers/accuracy.py

```diff
@@ -1,4 +1,38 @@
+from typing import Iterable, List
 from sklearn.metrics import r2_score as sk_r2_score
+from sklearn.metrics import f1_score as sk_f1_score
+from sklearn.metrics import recall_score as sk_recall_score
+from sklearn.metrics import precision_score as sk_precision_score
+
+
+def to_binary(y: Iterable) -> List[int]:
+    try:
+        y_binarized = []
+        for ele in y:
+            if str(ele).lower() == 'true':
+                y_binarized.append(1)
+            elif str(ele).lower() == 'false':
+                y_binarized.append(0)
+            else:
+                y_binarized.append(int(ele))
+
+        assert len(set(y_binarized)) < 3
+        assert 1 in y_binarized or 0 in y_binarized
```

Comment:
I feel like this should be "and" but I get the point which is there should be at least one instance of both "1" and "0".

Reply:
I think there's cases where we only predict 0 or 1 and this accounts for that.